cmd/eksctl/create.go

```diff
@@ -99,6 +99,7 @@ func createClusterCmd() *cobra.Command {
 
 	fs.BoolVar(&cfg.Addons.WithIAM.PolicyAmazonEC2ContainerRegistryPowerUser, "full-ecr-access", false, "enable full access to ECR")
 	fs.BoolVar(&cfg.Addons.Storage, "storage-class", true, "if true (default) then a default StorageClass of type gp2 provisioned by EBS will be created")
+	fs.BoolVar(&cfg.Addons.WithIAM.AutoScalingGroup, "auto-scaling-policy", false, "enable iam policy dependency for cluster-autoscaler")
```

Comment:
Please use 'IAM' in the description
```suggestion
	fs.BoolVar(&cfg.Addons.WithIAM.AutoScalingGroup, "auto-scaling-policy", false, "enable IAM policy dependency for cluster-autoscaler")
```